pyaerocom/ungriddeddata.py

```diff
@@ -1230,7 +1230,12 @@ def to_station_data_all(
                 out_data["stats"].append(data)
 
             # catch the exceptions that are acceptable
-            except (VarNotAvailableError, TimeMatchError, DataCoverageError) as e:
+            except (
+                VarNotAvailableError,
+                TimeMatchError,
+                DataCoverageError,
+                NotImplementedError,
+            ) as e:
```

Comment:
not sure if this lines are covered somewhere on the test suite, but it would be great to test that `NotImplementedError` can be used to filter obs

Reply:
Not sure how to test these exception effectively. Any ideas?

Reply:
The only ideas I have right now is a fake reader that throws these exceptions or a fabricated data set. Both is takes some time to create.

Reply:
I will extend these exceptions for the next release (e.g. with the exception raised when a station moved more than a kilometer).
How about postponing that test until then?